features/kiesrecht.feature

```diff
@@ -52,102 +23,99 @@ Rule: Aanduiding uitgesloten kiesrecht (38.10) wordt geleverd als boolean
+		| uitgesloten zonder einddatum       | 1                  |                       | 1               | persoon is uitgesloten  |                      |
+		| einddatum uitsluiting in toekomst  | 1                  | 20300101              | 1               | persoon is uitgesloten  | 20300101             |
+		| einddatum uitsluiting in verleden  | 1                  | 20220301              |                 |                         |                      |
+		| einddatum uitsluiting vandaag      | 1                  | 20220315              |                 |                         |                      |
+		| einddatum uitsluiting morgen       | 1                  | 20300316              | 1               | persoon is uitgesloten  | 20300316             |
+		| einddatum uitsluiting vorige maand | 1                  | 20220200              |                 |                         |                      |
+		| einddatum uitsluiting deze maand   | 1                  | 20220300              | 1               | persoon is uitgesloten  | 20220300             |
+		| einddatum uitsluiting vorig jaar   | 1                  | 20210000              |                 |                         |                      |
+		| einddatum uitsluiting dit jaar     | 1                  | 20220000              | 1               | persoon is uitgesloten  | 20220000             |
+		| einddatum uitsluiting onbekend     | 1                  | 00000000              | 1               | persoon is uitgesloten  | 00000000             |
+		| persoon ontvangt oproep            | 2                  |                       | 2               | persoon ontvangt oproep |                      |
 
-@proxy
+@gba
 Rule: Uitsluiting van kiesrecht wordt alleen opgenomen wanneer de einddatum uitsluiting in de toekomst ligt
 	- Wanneer alleen het jaar van de einddatum uitsluiting bekend is, dan wordt de uitsluiting opgenomen tot en met dat jaar.
 	- Wanneer het jaar en de maand van de einddatum uitsluiting bekend is, en de dag niet, dan wordt de uitsluiting opgenomen tot en met die maand.
 	- Een volledig onbekende einddatum uitsluiting wordt hetzelfde geïnterpreteerd en weergegeven als het niet aanwezig zijn van die datum.
 	- Wanneer einddatum uitsluiting geen waarde heeft en uitsluiting kiesrecht wel, wordt uitsluiting kiesrecht wel opgenomen.
 	- Wanneer einddatum uitsluiting in het verleden ligt of vandaag is, wordt het niet opgenomen.
 
-	@proxy
+	@gba
 	Abstract Scenario: kiesrecht <omschrijving>
 		Gegeven het systeem heeft een persoon met de volgende gegevens
 		| naam                        | waarde     |
 		| burgerservicenummer         | 999990001  |
-		En de persoon heeft de volgende kiesrecht gegevens
+		En de persoon heeft de volgende 'kiesrecht' gegevens
 		| naam                                  | waarde                  |
 		| uitgeslotenVanKiesrecht (38.10)       | <uitsluiting kiesrecht> |
```

Comment:
in het eerste scenario gebruik je in Gegeven de naam "aanduiding uitgesloten kiesrecht (38.10)" met een waarde "A". Hier gebruik je "uitgeslotenVanKiesrecht (38.10)" met de waarde true.

Ik denk dat dit in het eerste scenario goed is gedaan. Aangezien dit een @gba scenario is, moet je beschrijven hoe dit in bron voor @gba (dus GBA-V) staat.